build_kvm_image.sh

```diff
@@ -47,6 +47,11 @@ prepare_installer_disk()
 create_disk
 prepare_installer_disk
 
+# Force o.s. to drop cache so that KVM can get 2G memory
+free -m
+echo 1 | sudo tee /proc/sys/vm/drop_caches
```

Comment:
Just curious, is there a reason you chose to pipe to `tee` over simply `echo 1 > /proc/sys/vm/drop_caches`?

Reply:
sudo echo 1 >/proc file doesn't work if the current use is not root...

Reply:
emm. there is also a sudo bash -c "command" trick :-)